### DISCUSSION TITLE
Wouldn't it be nice to change between types of lists?

### DISCUSSION THREAD

**Discussion by wessleyn** [vkbo/novelWriter#2054] [Ideas]
Selecting a bullet list or dash list and turn it into a numbered list...as simple as that..

**Comment by vkbo** [comment]
novelWriter doesn't support lists.

**Reply by wessleyn** [reply]
> novelWriter doesn't support lists.

But ain't list a crucial part in novel planning, also the we do use dashes to represent  a list , so if you're considering stuff.  Just know 😄 this one would help me and perhaps others who vote on the thread


We don't ask for much just simple numbering for sentences

**Reply by vkbo** [reply]
novelWriter is not a planning/research tool. That's not what the notes are for. See #815.

Lists are supported indirectly since you can just type a dash and items on separate lines and it will look like a list, but they are not treated as ort formatted as lists.
